### PR TITLE
Fix NEMOH macOS

### LINKED ISSUE #150: NEMOH calls broken for macOS
Looks like the `NEMOH` interface stopped working on macOS at e7a75ef96afaebbe0647968d55bfb6b230a66300. Assuming that it is still working on Windows, I suspect this has to do with file systems (maybe file path separators?). It is a bit hard to debug since the changes with e7a75ef96afaebbe0647968d55bfb6b230a66300 were so significant.

I found one spot where it looks like backslashes have been used, this doesn't seem to be the culprit...
https://github.com/SNL-WaterPower/WecOptTool/blob/9610c3fbdf52a507324f22f4e0e1506de4bbf688/toolbox/%2BWecOptTool/%2Bsolver/NEMOH.m#L281

---
[test_results.pdf](https://github.com/SNL-WaterPower/WecOptTool/files/4891466/test_results.pdf)

```
MATLAB Version: 9.8.0.1396136 (R2020a) Update 3
MATLAB License Number: 10848
Operating System: Mac OS X  Version: 10.14.6 Build: 18G5033 
Java Version: Java 1.8.0_202-b08 with Oracle Corporation Java HotSpot(TM) 64-Bit Server VM mixed mode
```

### PR DESCRIPTION
**Pull request by ryancoe**
## Description
Fixes calls interface with `NEMOH` for macOS (see #150 for details)

Fixes #150
## Checklist:

- [x] Added the results of running the test suite (in pdf form)
- [x] All new files contain the GPL header
- [x] If `example.m` has been modified, the content / line numbers in 
  `docs\user\example.rst` are still valid or have been fixed

### CODE REVIEW

File: toolbox/+WecOptTool/+solver/nemohcalbody.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 --- Body %i ------------------------------------------------------------------
-%s ! Name of mesh file
+'%s' ! Name of mesh file
```
Comment (ryancoe) [inline toolbox/+WecOptTool/+solver/nemohcalbody.txt:2]: This was the bug:  the name of the mesh file listed in `Nemoh.cal` must have quotes.

Prior to a15a2ac: `./mesh/axisym_1.dat ! Name of mesh file`
Fixed with a15a2ac: `'./mesh/axisym_1.dat'		! Name of mesh file`

**Comment by ryancoe** [comment]
[test_results.pdf](https://github.com/SNL-WaterPower/WecOptTool/files/4891791/test_results.pdf)

@ssolson @H0R5E - please make that this doesn't break it for Windows

**Comment by H0R5E** [comment]
The removal of the double slash caused a problem with fprintf with trying to interpret "\m", but I was able to remedy that by printing the raw string line by line, as there are no special characters used apart from the newline at the end. Here are my tests:

[test_results.pdf](https://github.com/SNL-WaterPower/WecOptTool/files/4892166/test_results.pdf)


Check it works for you as well, and if so, go ahead and merge it.

**Comment by ryancoe** [comment]
Still good for me: [test_results.pdf](https://github.com/SNL-WaterPower/WecOptTool/files/4892632/test_results.pdf)

**Pull request merged**